ray-curator/pyproject.toml

```diff
@@ -65,6 +65,11 @@ video = [
     "opencv-python",
 ]
 
+all = [
+    "ray_curator[text]",
+    "ray_curator[video]",
```

Comment:
let's also add cuda12x because that's where the real check would be... I'm concerned since we don't have GPU tests we might not be able to test things thoroughly whether this docker image indeed works.. If anyone (@sarahyurick / @ayushdg ) have good ideas what to do in the mean time to ensure this docker image is good wrt rapids imports pls opine

Reply:
Yes, agree about the `cuda12x` dependencies. The only text GPU tests we have at the moment for Ray Curator are in https://github.com/NVIDIA-NeMo/Curator/pull/881, which I have run locally but not on GitHub.

We may want to merge `ray-api` into `main` soon, if there is no other work planned on `main` anyway... thoughts?